src/pkg/scan/report/summary.go

```diff
@@ -24,33 +24,67 @@ import (
 	"github.com/pkg/errors"
 )
 
+// CVESet defines the CVE whitelist with a hash set way for easy query.
```

Comment:
`CVESet` should be moved to /pkg/scan or un-exported.

Reply:
I'd prefer to keep it is here because it's close to the WithCVEWhitelist options although it is also used in the vulnerable hander. It's ok to refer it with the package `report` because it will cause influences to the "report" summary.

Reply:
`CVESet` here is also defined as an alias for the model `map[string]interface{}` defined in the whitelist for easy referring and appending utility method.

Reply:
CVESet is utility to represent the whitelist, we are confusing the boundary in terms of report .vs. whitelist

Reply:
@reasonerjt 

Actually `CVESet` can replace the general one `map[string]interface{}` in the whitelist package. 

If you care about the package, I can do the change to replace the original one. I did NOT want to change many packages originally.